proxy.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * The talkback proxy for e2e tests. When making a request to the API during testing,
+ * it tries to use the responses it previously saved in `/test/tapes` folder. If no
+ * response is found there, it makes an actual request, and saves the response for
+ * future use.
+ * This makes it possible for the e2e tests to run without internet, and makes the
+ * tests less flaky due to changes in the API or API data.
+ */
+const talkback = require('talkback')
+const path = require('path')
+const host = 'https://api.openverse.engineering/v1'
```

Comment:
Let's just catch everything going to the API, otherwise things like analytics (which isn't version-prefixed) will fall through.

```suggestion
const host = 'https://api.openverse.engineering'
```